graylog2-shared/src/main/java/org/graylog2/shared/stats/ThroughputStats.java

```diff
@@ -38,32 +37,24 @@
     public ThroughputStats() {
         this.currentThroughput = 0;
         this.throughputCounter = new Counter();
-        this.benchmarkCounter = new Counter();
-        this.streamThroughput = new AtomicReference<ConcurrentHashMap<String, Counter>>(new ConcurrentHashMap<String, Counter>());
-        this.currentStreamThroughput =  new AtomicReference<HashMap<String, Counter>>();
-
+        this.streamThroughput = new AtomicReference<>(new ConcurrentHashMap<String, Counter>());
+        this.currentStreamThroughput =  new AtomicReference<>();
     }
 
+    @Deprecated
     public long getCurrentThroughput() {
```

Comment:
No usages within the graylog2-server code base, so we could remove this instead of marking it as deprecated.